grpc/proto/v1/tenants.proto

```diff
@@ -16,7 +18,7 @@ enum TenantActivityStatus {
 
 message TenantsGetRequest {
   string collection = 1;
-  bool is_consistent = 2;
+  optional ConsistencyLevel consistency_level = 2;
```

Comment:
I would say considering the way it's used it makes sense to keep it a boolean. What do you think ? What was the reasoning behind changing this to a consistency level ?

Reply:
The reasoning is to align it with how the concept of consistency was used in previous versions. However, if this no longer makes sense generally (as above) then we should change the nomenclature that we use